### ISSUE TITLE
Add cake-contrib user

### ISSUE THREAD

**Issue by pascalberger** [cake-contrib/Cake.DotNetLocalTools.Module#1]
First of all, I wanted to thank you for adding to the Cake community by adding this addin.

I was just wondering if you are aware of [Cake Contrib organization](https://cakebuild.net/community/cake-contrib). Its main goal is to increase the visibility of addins, and also trying to ensure their long term maintainability.

To that end, we are asking addin creators to add the `cake-contrib` user on NuGet as a co-owner (this can be done through the NuGet website by clicking on Manage Owners on the package page).

Would you be interested in doing this? If you have any questions about this, please let me know. There was some initial concern that the Cake Team were trying to "take over" packages, and that couldn't be further from the truth, and if you have this concern, or others, I would like to address them.

Thanks!

**Comment by ap0llo** [comment]
Hi, thanks for reaching out.

I wasn't really aware of the fact that `cake-contrib` is intended to be a co-owner of all cake packages and thought that would just apply to packages which's source code is in the Cake Contrib organization.
Seeing that all the cake extensions I could find do this as well, I don't see any reason not to add the user - I support the idea of taking care of the extension ecosystem instead of solely relying on single contributors.

I added `cake-contrib` as a co-owner of the `Cake.DotNetLocalTools.Module` package as well as the addin `Cake.GitHubReleases` I recently published as well.

That being said, I think I would also like to transfer the GitHub repositories into the Cake Contrib organization.
However, I have some questions regarding this.
I opened up a new discussion for transferring the repos and my questions [here](https://github.com/cake-build/cake/discussions/3618)
